include/k4a/k4a.hpp

```diff
@@ -870,6 +921,28 @@ class transformation
         }
     }
 
+    /** Transforms depth map and a custom image into the geometry of the color camera.
+     * Throws error on failure
+     *
+     * \sa k4a_transformation_depth_image_to_color_camera_custom
+     */
+    std::pair<image, image>
+    depth_image_to_color_camera_custom(const image &depth_image,
+                                       const image &custom_image,
+                                       k4a_transformation_interpolation_type_t interpolation_type,
+                                       uint32_t invalid_custom_value) const
+    {
+        image transformed_depth_image;
+        image transformed_custom_image;
+        depth_image_to_color_camera_custom(depth_image,
+                                           custom_image,
+                                           &transformed_depth_image,
+                                           &transformed_custom_image,
+                                           interpolation_type,
+                                           invalid_custom_value);
+        return { transformed_depth_image, transformed_custom_image };
```

Comment:
>transformed_depth_image, transformed_custom_image  [](start = 17, length = 50)

nit: I'm pretty sure this results in the copy constructors getting invoked to make the std::pair, which ends up doing an addref/decref.
Consider wrapping these in std::move to dodge the refcount ++--.

Reply:
Is it <code>{std::move(transformed_depth_image), std::move(transformed_custom_image)}</code>?

Reply:
Yep!

---
In reply to: [311196260](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/pull/596#discussion_r311196260) [](ancestors = 311196260)

Reply:
Thanks,